### DISCUSSION TITLE
How can I break a line in a cell in a markdown table?

### DISCUSSION THREAD

**Discussion by xuejiao-joy** [facebook/docusaurus#9085] [Q&A]
How can I break a line in a cell in a markdown table? I‘ve tried `<br>` (which prompts an error while compiling the web), `<br/>` (which displays the content following the `<br/>` tag in another row), `</br>` (which doesn't work), and "\" (which also doesn't work)... 
I also tried HTML code to write the table... but the `<br>` tag also cannot be resolved....

**Comment by Josh-Cena** [comment]
> `<br/>` (which displays the content following the `<br/>` tag in another row)

This is supposedly the correct solution. I don't know why you would get another row, though. Do you have a working example to share?

**Reply by xuejiao-joy** [reply]
It worked. thank you soooo much~
It might be because I made some errors in other places that resulted another row.~
